worlds/mlss/Items.py

```diff
@@ -0,0 +1,199 @@
+    ItemData(77771109, "Blue Jeans", ItemClassification.useful, 0xCE),
+    ItemData(77771110, "Parasol Pants", ItemClassification.useful, 0xCF),
+    ItemData(77771111, "Hard Pants", ItemClassification.useful, 0xD0),
+    ItemData(77771112, "Heart Jeans", ItemClassification.useful, 0xD1),
+    ItemData(77771113, "Plaid Trousers", ItemClassification.useful, 0xD2),
+    ItemData(77771114, "#1 Trousers", ItemClassification.useful, 0xD3),
+    ItemData(77771115, "Safety Slacks", ItemClassification.useful, 0xD4),
+    ItemData(77771116, "Shroom Pants", ItemClassification.useful, 0xD5),
+    ItemData(77771117, "Shroom Bells", ItemClassification.useful, 0xD6),
+    ItemData(77771118, "Shroom Slacks", ItemClassification.useful, 0xD7),
+    ItemData(77771119, "Peachy Jeans", ItemClassification.useful, 0xD8),
+    ItemData(77771120, "Mushwin Pants", ItemClassification.useful, 0xD9),
+    ItemData(77771121, "Mushluck Pants", ItemClassification.useful, 0xDA),
+    ItemData(77771122, "Scandal Jeans", ItemClassification.useful, 0xDB),
+    ItemData(77771123, "Street Jeans", ItemClassification.useful, 0xDC),
+    ItemData(77771124, "Tropic Slacks", ItemClassification.useful, 0xDD),
+    ItemData(77771125, "Hermetic Pants", ItemClassification.useful, 0xDE),
+    ItemData(77771126, "Beanstar Pants", ItemClassification.useful, 0xDF),
+    ItemData(77771127, "Peasley Slacks", ItemClassification.useful, 0xE0),
+    ItemData(77771128, "Queen B. Jeans", ItemClassification.useful, 0xE1),
+    ItemData(77771129, "B. Brand Jeans", ItemClassification.useful, 0xE2),
+    ItemData(77771130, "Heart Slacks", ItemClassification.useful, 0xE3),
+    ItemData(77771131, "Casual Slacks", ItemClassification.useful, 0xE4),
+    ItemData(77771132, "Casual Coral", ItemClassification.useful, 0xEB),
+    ItemData(77771133, "Harhall's Pants", ItemClassification.useful, 0xF1),
+    ItemData(77771134, "Wool Trousers", ItemClassification.useful, 0xF3),
+    ItemData(77771135, "Iron Pants", ItemClassification.useful, 0xF7),
+    ItemData(77771136, "Greed Wallet", ItemClassification.useful, 0xF8),
+    ItemData(77771137, "Bonus Ring", ItemClassification.useful, 0xF9),
+    ItemData(77771138, "Excite Spring", ItemClassification.useful, 0xFA),
+    ItemData(77771139, "Great Force", ItemClassification.useful, 0xFB),
+    ItemData(77771140, "Power Grip", ItemClassification.useful, 0xFC),
+    ItemData(77771141, "Cobalt Necktie", ItemClassification.useful, 0xFD),
+    ItemData(77771142, "Game Boy Horror SP", ItemClassification.useful, 0xFE),
+    ItemData(77771143, "Dragohoho Defeated", ItemClassification.skip_balancing, 0x0),
+    ItemData(77771144, "Queen Bean Defeated", ItemClassification.skip_balancing, 0x0),
+    ItemData(77771145, "Chuckolator Defeated", ItemClassification.skip_balancing, 0x0),
+    ItemData(77771146, "Oasis Access", ItemClassification.skip_balancing, 0x0),
+    ItemData(77771147, "Mom Piranha Defeated", ItemClassification.skip_balancing, 0x0),
```

Comment:
These are locked items that correspond to event flags; They do not need to be classed as skip_balancing (since they don't affect progression balancing regardless). Also, these should probably be changed to Events rather than actual placed items, otherwise people could place them in starting inventory or plando them on places they shouldn't be.

Reply:
Theyre marked as skip balancing so they can be distinguised as non-filler, non-progression items. Also theyre used fpr the tracker, so they need to be seen by the server, which events cannot.

Reply:
skip balancing by itself shouldn't be used, progression_skip_balancing is the only common combination of skip balancing currently used. But from a quick look it seems like these aren't actually used in logic anywhere (and currently wouldn't be able to be) which would mean they shouldn't even be events.
That being said, if you only need these items to notify the tracker that something has changed you can use https://github.com/ArchipelagoMW/Archipelago/blob/main/docs/network%20protocol.md#set data storage keys instead which can then be read by the tracker, and an example in emerald's client
https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/pokemon_emerald/client.py#L344